protocols/CopyToResultsDir.sh

```diff
@@ -84,35 +84,35 @@ usedWorkflow=$(basename ${workflow})
 		echo "Skip insertSizeMetrics. seqType is: ${seqType}"
 	fi
 
-#copy variant to results directory.
-
-	cp "${intermediateDir}/${project}.variant.calls.genotyped.vcf" "${projectResultsDir}/variants/"
-	cp "${intermediateDir}/${project}.variant.calls.genotyped.vcf.md5" "${projectResultsDir}/variants/"
-
 # copy GeneCounts to results directory
 
-	cp "${intermediateDir}"/*.counts.txt "${projectResultsDir}/expression/perSampleExpression"
-	cp "${projectHTseqExpressionTable}" "${projectResultsDir}/expression/expressionTable"
-	cp "${annotationGtf}" "${projectResultsDir}/expression/"
+	cp "${intermediateDir}"/*.counts.txt "${projectResultsDir}"/expression/
+	cp "${annotationGtf}" "${projectResultsDir}"/expression/
+	cp "${projectHTseqExpressionTable}" "${projectResultsDir}"/expression/
```

Comment:
i would quote the complete path:
`"${projectResultsDir}/expression/"` instead of `"${projectResultsDir}"/expression/`
this is also for lines 89 and 90